dkrap.py

```diff
@@ -46,3 +72,16 @@ def tinyVerse():
 chorus()
 tinyVerse()
 chorus()
+lankyVerse()
+chorus()
+diddyVerse()
+chorus()
+chunkyVerse()
+chorus()
+#im not writing a function for this one line
```

Comment:
what line u know u can comment on the line after the line ur referring to is donezo

(obvi i know what line it is but cmon son)